<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grimeCleaner</class>
 <widget class="QMainWindow" name="grimeCleaner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grime Cleaner Pro</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="malwarebytesButton">
       <property name="text">
        <string>Open Malwarebytes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="spybotButton">
       <property name="text">
        <string>Open Spybots</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ccleanerButton">
       <property name="text">
        <string>Open Ccleaner</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="diskcleanupButton">
       <property name="text">
        <string>Launch Disk Clean Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prefetchButton">
       <property name="text">
        <string>Clean Prefetch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="installsoftwareButton">
       <property name="text">
        <string>Install All Software</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>361</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGrime_Cleaner_Pro">
    <property name="title">
     <string>Grime Cleaner Pro</string>
    </property>
   </widget>
   <addaction name="menuGrime_Cleaner_Pro"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
